homeassistant/components/ihc/switch.py

```diff
@@ -62,7 +66,7 @@ def __init__(
         self._state = False
 
     @property
-    def is_on(self):
+    def is_on(self) -> bool:
```

Comment:
This is type hint improvement - unrelated to config-flow. Please move to preliminary/follow-up PR.

Reply:
Done in PR [90350](https://github.com/home-assistant/core/pull/90350)